### PR TITLE
Add Streamlit Webapp

### PR DESCRIPTION
**Pull request by rohitjoshi6**
# Description

This is a python script for creating a webapp using streamlit. It is a simple compound-interest calculator.

Fixes #31 

## Type of change

Choosing one or more options from the following as per the nature of your Pull request.
- NOTE: *These boxes can be checked using **`[X]`***

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Documentation Update

# Checklist:
Please tick all the boxes that are fulfilled by your Pull Request.

- [x] I have named my files and folder, according to this project's guidelines.
- [x] My code follows the style guidelines of this project.
- [x] I have commented on my code, particularly in hard-to-understand areas.
- [x] My Pull Request has a descriptive title. (not a vague title like `Update index.md`)
- [x] I have created a helpful and easy to understand `README.md`, according to the given [`README_TEMPLATE.`](https://github.com/Python-World/Python_and_the_Web/blob/master/README_TEMPLATE.md)
- [x] I have included a requirements.txt file (if external libraries are required.)
- [x] My changes do not produce any warnings.
- [x] I have added a working sample/screenshot of the script.

### CODE REVIEW

**Comment by AdityaJ7** [comment]
Uncommit your image scraper from this

**Comment by rohitjoshi6** [comment]
> Uncommit your image scraper from this

Can i just know how can i do that plz??

**Comment by AdityaJ7** [comment]
> > Uncommit your image scraper from this
> 
> Can i just know how can i do that plz??

I'll do it

**Comment by rohitjoshi6** [comment]
> > > Uncommit your image scraper from this
> > 
> > 
> > Can i just know how can i do that plz??
> 
> I'll do it

ok thanks alot! I appreciate it!! But can u tell me what command is used so that i can learn it?

**Comment by AdityaJ7** [comment]
@rohitjoshi6, just go to your forked repo and remove the unwanted files, then push the changes

**Comment by rohitjoshi6** [comment]
> @rohitjoshi6, just go to your forked repo and remove the unwanted files, then push the changes

Ok got it!! Thanks alot

**Comment by AdityaJ7** [comment]
> > @rohitjoshi6, just go to your forked repo and remove the unwanted files, then push the changes
> 
> Ok got it!! Thanks alot

Remove the extra spaces from the code

File: Scripts/Miscellaneous/Streamlit_webapp/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+# Pre-requisites:
+
+Install the required modules:
+```bash
+pip install -r requirements.txt
+```
+# Instructions to run this script:
+
+```bash
+python Compound_interest.py
```
Comment (AdityaJ7) [inline Scripts/Miscellaneous/Streamlit_webapp/README.md:13]: Stremlit apps are not executed like this

it should be streamlit run Compound_interest.py

File: Scripts/Miscellaneous/Streamlit_webapp/Compound_interest.py [hunk 2]
```diff
@@ -0,0 +1,84 @@
+import streamlit as st
+
+import pandas as pd
+
+# from sklearn import datasets
```
Comment (AdityaJ7) [inline Scripts/Miscellaneous/Streamlit_webapp/Compound_interest.py:3]: Remove the commented stuff

File: Scripts/Miscellaneous/Streamlit_webapp/Compound_interest.py [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+    st.write(comp_text)
+    data_1 = {'Computed_Compound_Interest': comp_text}
+
+    result = pd.DataFrame(data_1, index=[0])
+
+    return result
+
+
+
+
```
Comment (AdityaJ7) [inline Scripts/Miscellaneous/Streamlit_webapp/Compound_interest.py]: Minimise the spaces

**Review by AdityaJ7 — CHANGES_REQUESTED** [review]

**Comment by rohitjoshi6** [comment]
I have made the requested changes.

**Comment by AdityaJ7** [comment]
> I have made the requested changes.

I'll suggest you to run your scripts through flake8 module so that it approves they follow PEP-8 guidelines

**Comment by rohitjoshi6** [comment]
> > I have made the requested changes.
> 
> I'll suggest you to run your scripts through flake8 module so that it approves they follow PEP-8 guidelines

I will take care of it from now on. Thanks for the valuable suggestion!

**Review by AdityaJ7 — APPROVED** [review]

**Comment by AdityaJ7** [comment]
@all-contributors please add @rohitjoshi6 for code

**Pull request merged**